src/Components/Pages.tsx

```diff
@@ -107,33 +113,38 @@ function makeEmptyScene(): (routeMatch: SceneRouteMatch) => EmbeddedScene {
 }
 
 export function makeBreakdownPage(
-  routeMatch: SceneRouteMatch<{ service: string; label?: string }>,
+  routeMatch: RouteMatch,
   parent: SceneAppPageLike,
   slug: ParentDrilldownSlugs
 ): SceneAppPage {
-  const { service } = extractServiceFromRoute(routeMatch);
+  const { labelName, labelValue } = extractValuesFromRoute(routeMatch);
   return new SceneAppPage({
     title: slugToBreadcrumbTitle(slug),
     layout: PageLayoutType.Custom,
-    url: ROUTES[slug](service),
+    url: ROUTES[slug](labelValue, labelName),
     preserveUrlKeys: DRILLDOWN_URL_KEYS,
     getParentPage: () => parent,
     getScene: (routeMatch) => getServicesScene(routeMatch),
   });
 }
 
 export function makeBreakdownValuePage(
-  routeMatch: SceneRouteMatch<{ service: string; label: string }>,
+  routeMatch: RouteMatch,
   parent: SceneAppPageLike,
   slug: ChildDrilldownSlugs
 ): SceneAppPage {
-  const { service } = extractServiceFromRoute(routeMatch);
-  const { label } = extractLabelNameFromRoute(routeMatch);
+  const { labelName, labelValue, breakdownLabel } = extractValuesFromRoute(routeMatch);
+
+  if (!breakdownLabel) {
```

Comment:
Do we gracefully fail here? We can definitely log the error, but hopefully we should take the user back to the service selection page. Basically, anything but a broken page with no explanation.

Reply:
I'm not able to repro throwing this error at all, if there is no breakdownLabel in the URL it won't route to this endpoint